code/main.cpp

```diff
@@ -1,7 +1,6 @@
-#include "hwlib.hpp"
-#include "test_arm_c.h"
-#include "robot_arm_interface_c.h"
-#include "test_arm_c.h"
+#include <hwlib.hpp>
+#include <../headers/robot_arm_interface_c.hpp>
```

Comment:
This should move to a different branch as well and you dont need to add the folder to the include if the makefile is correct

"../headers" <-- This part